### ISSUE TITLE
What is api.minecraft.services.com?

### ISSUE THREAD

**Issue by Pastetas** [Schubilegend/SchubiAuthV2#1]
When u load this mod, it grabs ur token and sends it to this api, my question is why? and what is that api? and what does it do with my token there?

**Comment by BattleB0t** [comment]
No way, I knew it was a rat. Thanks for telling me (not)

**Comment by schubilegend** [comment]
1 google search requests or one question to chatgpt would also have solved your retarded question. please kys

**Comment by Majanito** [comment]
Hello,

would be cool if you actually read the code instead of complaining and doing a bit of research.

"https://api.minecraftservices.com/minecraft/profile" is the official Mojang API endpoint for fetching a users profile information with a session token as authorization. For more info on this endpoint, please refer to the official Mojang API Documentation™ here:

https://wiki.vg/Mojang_API#Profile_Information

Also, your statement that the mod grabs your token on load is factually wrong. If you'd actually take a minute to read the code, you would noticed that the method is only called once you login with a session token, as you can see here:

https://github.com/Schubilegend/SchubiAuthV2/blob/main/src/main/java/dev/schubilegend/GUI/SessionGUI.java

This is the only way to reliably get the username and the UUID of a player using their session token, without involving any untrusted third partys.

So in conclusion, the session id is only sent to mojangs official servers and it doesn't send your own session token, just the ones you login to with this mod. 

Please do not open any useless issues in the future, as this achieves nothing except wasting my time.

**Comment by BattleB0t** [comment]
Wouldn’t trust majanito he‘s big rattor
